### ISSUE TITLE
Wrong Status Code Description

### ISSUE THREAD

**Issue by c4s4** [long2ice/swagin#6]
Status code description is duplicated on all codes as shown in screenshot below:

![Capture d’écran_2022-02-08_11-20-04](https://user-images.githubusercontent.com/1253887/152967977-e45e8e7c-095c-4fcc-b28e-b2206be51022.png)

Router is defined as follows:

```go
var DeployProject = router.New(
	&DeployProjectInput{},
	router.Summary("Deploy Project"),
	router.Description("Deploy project for given configuration"),
	router.Responses(router.Response{
		"204": router.ResponseItem{
			Model:       model.Created{},
			Description: "Project created",
		},
		"400": router.ResponseItem{
			Model:       model.BadRequest{},
			Description: "Bad request",
		},
		"404": router.ResponseItem{
			Model:       model.NotFound{},
			Description: "Not found",
		},
		"500": router.ResponseItem{
			Model:       model.InternalServerError{},
			Description: "Error response",
		},
	}),
)
```

**Comment by c4s4** [comment]
This can be fixed in *swagger/swagger.go* with following code:

```go
func (swagger *Swagger) getResponses(response router.Response) openapi3.Responses {
	ret := openapi3.NewResponses()
	for k, v := range response {
		schema := swagger.getResponseSchemaByModel(v.Model)
		content := openapi3.NewContentWithJSONSchema(schema)
		description := v.Description
		ret[k] = &openapi3.ResponseRef{
			Value: &openapi3.Response{
				Description: &description,
				Content:     content,
				Headers:     v.Headers,
			},
		}
	}
	return ret
}
```

**Comment by long2ice** [comment]
Why this?

**Comment by c4s4** [comment]
Here is an explanation of this issue: https://stackoverflow.com/questions/29923956/golang-reusing-memory-address-copying-from-slice

**Comment by long2ice** [comment]
Get it, thanks!

**Comment by c4s4** [comment]
Fixed with PR #7
